src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -218,13 +237,12 @@ public static class Validator
         public static void ValidateProperty(object value, ValidationContext validationContext)
         {
             // Throw if value cannot be assigned to this property.  That is not a validation exception.
-            var propertyType = _store.GetPropertyType(validationContext);
+            Type propertyType = s_store.GetPropertyType(validationContext);
             EnsureValidPropertyType(validationContext.MemberName, propertyType, value);
 
-            var attributes = _store.GetPropertyValidationAttributes(validationContext);
+            IEnumerable<ValidationAttribute> attributes = s_store.GetPropertyValidationAttributes(validationContext);
 
-            GetValidationErrors(value, validationContext, attributes, false).FirstOrDefault()
-                ?.ThrowValidationException();
+            GetValidationErrors(value, validationContext, attributes).FirstOrDefault()?.ThrowValidationException();
```

Comment:
Why leaving out 4th arg is ok?
I would separate such change from code style-only changes at minimum.